src/tasks/Tasks.js

```diff
@@ -0,0 +1,149 @@
+import React, { useEffect, useState } from 'react'
+import TasksForm from './TasksForm'
+import Popup from '../components/Popup';
+import { Paper, makeStyles, Toolbar, InputAdornment, Card, Typography, CardContent } from '@material-ui/core';
+import Button from '../components/Button';
+import * as taskService from '../services/taskService';
+import { Search } from "@material-ui/icons";
+import SearchBox from '../components/SearchBox';
+import AddIcon from '@material-ui/icons/Add';
+import PageHeader from '../components/PageHeader';
+import AssignmentIcon from '@material-ui/icons/Assignment';
+import TaskCard from '../components/TaskCard';
+import DisplayMap from '../components/DisplayMap';
+
+const useStyles = makeStyles(theme => ({
+  pageContent: {
+    margin: theme.spacing(5),
+    padding: theme.spacing(3)
+  },
+  searchInput: {
+    width: '75%'
+  },
+  newButton: {
+    position: 'absolute',
+    right: '10px'
+  },
+  cardWrapper: {
+    display: 'flex',
+    flexDirection: 'row',
+    flexWrap: 'wrap',
+    gap: '15px',
+    marginLeft: '38px'
+  },
+}))
+
+export default function Tasks() {
+  const classes = useStyles();
+  const [records, setRecords] = useState(taskService.getAllTasks());
+  const [openPopup, setOpenPopup] = useState(false);
+  const [recordForEdit, setRecordForEdit] = useState(null);
+  const [filterFn, setFilterFn] = useState({ fn: items => items });
+  
+
+ 
+
+   const addOrEdit = (task, resetForm) => {
```

Comment:
no need to pass resetForm.  
If you want to reset the form after calling the function, do this in the caller function.  
```
// somwhere
addOrEdit(data)
resetForm()
```